Alloy/commands/compile/compilerUtils.js

```diff
@@ -4,7 +4,7 @@ var U = require('../../utils'),
 	os = require('os'),
 	fs = require('fs-extra'),
 	walkSync = require('walk-sync'),
-	jsonlint = require('jsonlint'),
+	jsonlint = require('@prantlf/jsonlint'),
```

Comment:
Does jsonlint provide anything extra over just a normal `JSON.parse`, we're not exactly parsing large objects so I don't think speed matters, but maybe the better errors are helpful?

Reply:
good point. I've looked at the git blame and found this one:
https://github.com/tidev/alloy/commit/69008a4ba996fdab889111856e7cc9b0f6d2f98c
>  major refactor and logging improvements to compile process, using jsonlint for JSON to get line number on errors
it was using the normal JSON.parse() before

Reply:
Yeah looking at the errors properly they do seem to be better by providing some context

```text
[ERROR] Error while adding widget dependency to config.json in "/Users/awam/Documents/titanium-workspace/test-app/app/config.json"
[ERROR] Parse error on line 10, column 21:
[ERROR] ...        "test": "1.0    },    "backbon...
[ERROR] -----------------------^
[ERROR] Unexpected token 
[ERROR] 
```

vs

```text
[ERROR] Error while adding widget dependency to config.json in "/Users/awam/Documents/titanium-workspace/test-app/app/config.json"
[ERROR] Unexpected token 
[ERROR]  in JSON at position 197
```